.travis.yml

```diff
@@ -1,26 +0,0 @@
-language: ruby
-
-addons:
-  apt:
-    packages:
-      - libcurl4-openssl-dev
-
-cache: bundler
-
-before_install:
-  - "echo 'gem: --no-document' > ~/.gemrc"
-  - gem update --system
-  - gem install bundler
-
-script:
-  - if [ "$TRAVIS_EVENT_TYPE" != "cron" ]; then bundle exec rake; fi
-  - if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then bundle exec thor updates:check --github-token $GH_TOKEN --upload; fi
```

Comment:
The `thor updates:check` runs once a month and can probably be added later once the deployment is working again.

Reply:
Agreed.  I'll add a second action to trigger those on the current schedule.